data/population_weights_via_fips.json

```diff
@@ -0,0 +1 @@
 0.0112, "53031": 0.0042, "53033": 0.2958, "53035": 0.0357, "53037": 0.0063, "53039": 0.0029, "53041": 0.0106, "53043": 0.0014, "53045": 0.0088, "53047": 0.0055, "53049": 0.003, "53051": 0.0018, "53053": 0.1188, "53055": 0.0023, "53057": 0.017, "53059": 0.0016, "53061": 0.108, "53063": 0.0687, "53065": 0.006, "53067": 0.0382, "53069": 0.0006, "53071": 0.008, "53073": 0.0301, "53075": 0.0066, "53077": 0.0329, "53999": NaN, "54001": 0.0092, "54003": 0.0665, "54005": 0.012, "54007": 0.0078, "54009": 0.0122, "54011": 0.0513, "54013": 0.004, "54015": 0.0047, "54017": 0.0047, "54019": 0.0237, "54021": 0.0044, "54023": 0.0065, "54025": 0.0193, "54027": 0.0129, "54029": 0.0161, "54031": 0.0077, "54033": 0.0375, "54035": 0.0159, "54037": 0.0319, "54039": 0.0994, "54041": 0.0089, "54043": 0.0114, "54045": 0.0179, "54047": 0.0098, "54049": 0.0313, "54051": 0.017, "54053": 0.0148, "54055": 0.0328, "54057": 0.015, "54059": 0.0131, "54061": 0.0589, "54063": 0.0074, "54065": 0.01, "54067": 0.0137, "54069": 0.0231, "54071": 0.0039, "54073": 0.0042, "54075": 0.0046, "54077": 0.0187, "54079": 0.0315, "54081": 0.0409, "54083": 0.016, "54085": 0.0053, "54087": 0.0076, "54089": 0.007, "54091": 0.0093, "54093": 0.0038, "54095": 0.0048, "54097": 0.0135, "54099": 0.022, "54101": 0.0045, "54103": 0.0084, "54105": 0.0032, "54107": 0.0466, "54109": 0.0114, "54999": NaN, "55001": 0.0035, "55003": 0.0027, "55005": 0.0078, "55007": 0.0026, "55009": 0.0454, "55011": 0.0022, "55013": 0.0026, "55015": 0.0086, "55017": 0.0111, "55019": 0.006, "55021": 0.0099, "55023": 0.0028, "55025": 0.0939, "55027": 0.0151, "55029": 0.0048, "55031": 0.0074, "55033": 0.0078, "55035": 0.018, "55037": 0.0007, "55039": 0.0178, "55041": 0.0015, "55043": 0.0088, "55045": 0.0063, "55047": 0.0032, "55049": 0.0041, "55051": 0.001, "55053": 0.0035, "55055": 0.0146, "55057": 0.0046, "55059": 0.0291, "55061": 0.0035, "55063": 0.0203, "55065": 0.0029, "55067": 0.0033, "55069": 0.0047, "55071": 0.0136, "55073": 0.0233, "55075": 0.0069, "55077": 0.0027, "55078": 0.0008, "55079": 0.1624, "55081": 0.0079, "55083": 0.0065, "55085": 0.0061, "55087": 0.0323, "55089": 0.0153, "55091": 0.0013, "55093": 0.0073, "55095": 0.0075, "55097": 0.0122, "55099": 0.0023, "55101": 0.0337, "55103": 0.003, "55105": 0.0281, "55107": 0.0024, "55109": 0.0156, "55111": 0.0111, "55113": 0.0028, "55115": 0.007, "55117": 0.0198, "55119": 0.0035, "55121": 0.0051, "55123": 0.0053, "55125": 0.0038, "55127": 0.0178, "55129": 0.0027, "55131": 0.0234, "55133": 0.0694, "55135": 0.0088, "55137": 0.0042, "55139": 0.0295, "55141": 0.0125, "55999": NaN, "56001": 0.0672, "56003": 0.0204, "56005": 0.0801, "56007": 0.0256, "56009": 0.0239, "56011": 0.0131, "56013": 0.0678, "56015": 0.0228, "56017": 0.0076, "56019": 0.0146, "56021": 0.1719, "56023": 0.0343, "56025": 0.138, "56027": 0.0041, "56029": 0.0504, "56031": 0.0145, "56033": 0.0527, "56035": 0.017, "56037": 0.0732, "56039": 0.0405, "56041": 0.0349, "56043": 0.0135, "56045": 0.012, "56999": NaN}
```

Comment:
I explicitly wanted to start off with them persisted because I want the snapshot to snapshot stability, even at the cost of some staleness. They are very easy to calculate, but I want them to not be sensitive to county reporting noise. The code that calculates them should be committed, with some helper function that gives you the option of overwriting the defaults, but I want to get away from tracking down run to run variations. Do you think there's a better middle way?